packages/headless/src/controllers/relevance-inspector/headless-relevance-inspector.test.ts

```diff
@@ -100,17 +117,20 @@ describe('RelevanceInspector', () => {
     engine = buildMockSearchAppEngine({state});
     relevanceInspector = buildRelevanceInspector(engine);
 
-    expect(relevanceInspector.state).toEqual({
-      isEnabled: true,
-      rankingInformation: rankingInformationSelector(engine.state),
-      executionReport: responseWithDebug.executionReport,
-      expressions: {
-        basicExpression: responseWithDebug.basicExpression,
-        advancedExpression: responseWithDebug.advancedExpression,
-        constantExpression: responseWithDebug.constantExpression,
-      },
-      userIdentities: responseWithDebug.userIdentities,
-      rankingExpressions: responseWithDebug.rankingExpressions,
-    });
+    expect(relevanceInspector.state).toMatchObject(
+      expect.objectContaining({
```

Comment:
And why does adding new property should fail the test ? 🤔  It's not breaking to do so. 

It's still going to fail if something that should be there is removed, however.

Reply:
Adding a property is not breaking, but if the test is not updated, and this new property is later removed, it will be a breaking change but the test will not catch it to alert the dev.